server/src/main/java/org/apache/calcite/server/ServerDdlExecutor.java

```diff
@@ -154,6 +156,14 @@ static Pair<CalciteSchema, String> schema(CalcitePrepare.Context context,
     return Pair.of(schema, name);
   }
 
+  /** Return the table from the given {@code} context and {@code} name. */
+  static Table table(CalcitePrepare.Context context, SqlIdentifier name) {
```

Comment:
Used only once.

It doesn't seem necessary to abstract it as a method.

Reply:
Deleted.